linkcheck/htmlutil/linkparse.py

```diff
@@ -175,10 +175,10 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
-        if tag =="a" and attrs.get_true('class', u''):
-           if ("broken_link" in attrs.get('class')):
-               log.debug(LOG_CHECK," Found a broken_link to %s, not considering further", attrs.get('href'))
-               return
+         if tag == "a" and attrs.get('class'):
+            if ("broken_link" in attrs.get('class').split()):
```

Comment:
This indentation change looks correct -- I didn't notice the short 3-space indent in my earlier review.  This change corrects it to the usual 4-space indent recommended by most Python style guides.